tests/system/projects/assets/kflow.py

```diff
@@ -41,22 +41,22 @@ def kfpipeline(model_pkg_class=default_pkg_class, build=0):
     )
 
     # train the model using a library (hub://) function and the generated data
-    train = funcs["train"].as_step(
+    train = funcs["auto_trainer"].as_step(
```

Comment:
Can you add a comment that the default handler is `train`? it wasn't clear to me at 1st sight